summary.php

```diff
@@ -188,14 +188,14 @@
 	<title>Resources Module - <?php echo $pageTitle; ?></title>
 	<link rel="stylesheet" href="css/style.css" type="text/css" media="print" />
 	<link rel="stylesheet" href="css/style.css" type="text/css" media="screen" />
-	<link rel="SHORTCUT ICON" href="images/butterflyfishfavicon.ico" />
+	<link rel="SHORTCUT ICON" href="images/favicon.ico" />
 
 	</head>
 	<body>
 
 
 	<div class='printContent'>
-	<table class='linedFormTable'>
+	<table class='linedFormTable' style='width:510px;'>
```

Comment:
Is there a specific reason why the width is added as an inline style instead of applied as an additional class? Or included in class 'linedFormTable'?